src/main/resources/locale_fr.properties

```
@@ -237,7 +237,8 @@ command.roles.info.message.reputationrole=Rôles de réputation
 command.roles.info.message.roleinfo=Info sur les rôles
 command.roles.list.description=Lister les rôles de réputation.
 command.roles.refresh.description=Mettre à jour les rôles des utilisateurs
-command.roles.refresh.message.finished=Mise à jour du rôle de l'utilisateur effectuée.
+command.roles.refresh.message.finished=Mise à jour du rôle de l'utilisateur effectuée.\nVérifié : %CHECKED%\nMis à jour : %UPDATED%\n
+command.roles.refresh.message.progress=**Refresh Progress**\n Vérifié : %CHECKED%\nMis à jour : %UPDATED%\n
```

DeepL suggestion and remove trailing newlines:
```suggestion
command.roles.refresh.message.finished=Mise à jour du rôle de l'utilisateur effectuée.\nVérifié: %CHECKED%\nMis à jour: %UPDATED%
command.roles.refresh.message.progress=**Progression de l'actualisation**\n Vérifié: %CHECKED%\nMis à jour: %UPDATED%
```